lix/README.md

```diff
@@ -1 +1,68 @@
-TODO
\ No newline at end of file
+# Lix (previously Project Lisa)
+
+Lix is the git compatible data layer for inlang apps.
+
+We build lix to bring developers and non technical editors together without sacrificing the power of full version control.
+
+We will also support other usecases for many applications with a
+"create > review > automate" flow:
+
+Examples for great lix apps (some require a later binary support layer):
+  - markdown editors
+  - bookkeeping
+  - editors for manuals
+  - figma files
+  - versioned spreadsheets
+  - CAD
+  - jupyter notebooks or linked datasets
+  - colocated bug tickets in an sqlite db in the repo
+
+##  Design Principles
+
+### 1. File-based.
+
+#### The filesystem is the universal data API.
+
+If apps A (e.g. CLI) and B (e.g. IDE) operate on files, both apps have interop.
+Read a file, write a file, simple. The moment project lix breaks this
+constraint, interop is lost. This Loom video illustrates the benefits of "lix
+operates on files" https://www.loom.com/share/f3beba7d8ee444c5bc0dde8b5ad41624.
+
+### 2. Git compatible.
+
+#### Without git compatibility, no adoption.
+
+Git is the most widely used version control system. If project lix is not git
+compatible, the wide ecosystem of git repositories can't be leveraged, and
+inlang wouldn't exist.
+
+### 3. Host Agnostic.
+
```

Comment:
I vote for leaving the design principles in a separate markdown file. The readme is mostly marketing purposes while the design principles is an engineering guideline, free from marketing. 

The design principles don;'t have to live in source-code.